src/classlibnative/bcltype/number.cpp

```diff
@@ -134,8 +135,191 @@ unsigned int Int64DivMod1E9(unsigned __int64* value)
 
 #else // _TARGET_X86_ && !FEATURE_PAL
 
-#pragma warning(disable:4273)
-extern "C" char* __cdecl _ecvt(double, int, int*, int*);
+void ecvt( double value, int count, int* dec, int* sign, wchar_t* digits )
```

Comment:
That would probably work. Although, just calling it _ecvt everywhere should too (this might require a minor cleanup in the PAL layer where the other non-windows implementation of _ecvt is)